input/resources/StructureDefinition-profile-patient.json

```diff
@@ -44,6 +44,216 @@
+        "fixedString": "Patient Medicaid number"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "https://signalbhn.org/fhir/sid/us-medicaidid"
+      },
+      {
+        "id": "Patient.identifier:MedicareID",
+        "path": "Patient.identifier",
+        "sliceName": "MedicareID",
+        "short": "Medicare identifier for this patient",
+        "definition": "Medicare identifier for this patient.",
+        "max": "1"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "Medicare ID"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.type.text",
+        "path": "Patient.identifier.type.text",
+        "fixedString": "Patient Unique Medicare Number"
+      },
+      {
+        "id": "Patient.identifier:MedicareID.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "http://hl7.org/fhir/sid/us-mbi"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId",
+        "path": "Patient.identifier",
+        "sliceName": "ProviderClientId",
+        "short": "ProviderClient identifier for this patient",
+        "definition": "ProviderClient identifier for this patient.",
+        "max": "1"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "MR"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "Medical Record Number"
+      },
+      {
+        "id": "Patient.identifier:ProviderClientId.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "PROVIDER SYSTEM URL"
+      },
+      {
+        "id": "Patient.identifier:ClientId",
+        "path": "Patient.identifier",
+        "sliceName": "ClientId",
+        "short": "Client identifier for this patient",
+        "definition": "Client identifier for this patient.",
+        "max": "1"
+      },
+      {
+        "id": "Patient.identifier:ClientId.type.coding.system",

```

Comment:
Patient.identifier:ClientId represents Signal's Beacon generated number, correct?   I don't think it fits in with anything in the **type.coding.system**, so I suggest removing all entries for type.coding--leave it blank for this one.

Reply:
Done

Reply:
@KasiSribal - Sorry to reneg on this.  I was reading through https://www.hl7.org/fhir/valueset-identifier-type.html, and I do think we should call it a "MR" code.  Would you add back the applicable values? i.e.
type.coding.system = https://www.hl7.org/fhir/valueset-identifier-type.html
type.coding.code = MR
type.coding.display = Medical record number